app/pods/agendas/template.hbs

```diff
@@ -26,7 +26,9 @@
 </Auk::Navbar>
 {{outlet}}
 {{#if this.isCreatingNewSession}}
-  <Meeting::NewMeetingModal
+  <Meeting::EditMeetingModal
+    @isNew={{true}}
```

Comment:
Maybe better to embed this logic in `Meeting::EditMeetingModal` by adding a tracked property `isNew` in the meeting modal component which is initialized in it's constructor based on `this.args.meeting.isNew`. Keeps the component interface simpler.